libraries/botbuilder-schema/botbuilder/schema/teams/_models_py3.py

```diff
@@ -2370,3 +2370,85 @@ def __init__(self, *, value=None, **kwargs) -> None:
 
     def _custom_init(self):
         return
+
+
+class MeetingDetails(Model):
+    """Specific details of a Teams meeting.
+
+    :param id: The meeting's Id, encoded as a BASE64 string.
+    :type id: str
+    :param ms_graph_resource_id: The MsGraphResourceId, used specifically for MS Graph API calls.
+    :type ms_graph_resource_id: str
+    :param scheduled_start_time: The meeting's scheduled start time, in UTC.
+    :type scheduled_start_time: str
+    :param scheduled_end_time: The meeting's scheduled end time, in UTC.
+    :type scheduled_end_time: str
+    :param join_url: The URL used to join the meeting.
+    :type join_url: str
+    :param title: The title of the meeting.
+    :type title: str
+    :param type: The meeting's type.
+    :type type: str
+    """
+
+    _attribute_map = {
+        "id": {"key": "uniqueId", "type": "str"},
+        "ms_graph_resource_id": {"key": "msGraphResourceId", "type": "str"},
+        "scheduled_start_time": {"key": "scheduledStartTime", "type": "str"},
+        "scheduled_end_time": {"key": "scheduledEndTime", "type": "str"},
```

Comment:
Note: msrest didn't seem to deserialize this into datetime, even when I set `"type": "datetime"`. I wasn't able to find a way to accomplish this, but other areas of the SDK also just deserialize into a `str`.

Reply:
I went with 'string' too.